app/services/converters/order_detail_to_journal_rows.rb

```diff
@@ -0,0 +1,23 @@
+module Converters
+  class OrderDetailToJournalRows
+
+    attr_accessor :journal, :order_detail, :total
+
+    def initialize(journal, order_detail, options = {})
+      @journal = journal
+      @order_detail = order_detail
+      @total = options[:total] || order_detail.total
+    end
+
+    def to_a
+      [{
+        account: order_detail.product.account,
```

Comment:
That makes sense for now. We can deal with some of the attribute merging/whatnot for when we get to the NU piece.

I would lean towards either changing the name of the class to something ending with `JournalRowAttributes` if we're not actually going to be returning JournalRows.